java/INSTALL

```diff
@@ -103,18 +108,25 @@ general overview of OpenDDS.
      use paths with no spaces (true on Unix too but spaces are rarely seen).
      The top level of the OpenDDS distribution is a directory that contains
      the subdirectory named dds.
+     
+     
+  3. Find the full path to JDK
+  
+     Linix    => export JAVA_HOME=$(readlink -f /usr/bin/javac | sed "s:/bin/javac::")
```

Comment:
If the user has supplied "--java" on the command line (with no =), I think there are a few cases we can handle (regardless of platform):

1. User has set JAVA_HOME in the environment -> use it
2. javac is found on PATH -> use location of bin/javac (after resolving symlinks on Unix) as JAVA_HOME
3. Error -> message about supplying full path to the JDK installation or setting up PATH to include javac